praw/models/reddit/multi.py

```diff
@@ -1,6 +1,7 @@
 """Provide the Multireddit class."""
 from json import dumps
 import re
+from typing import Any, Dict, List, Optional, TypeVar, Union
```

Comment:
Can we be consistent about `import ...` compared to `from ....` imports please?

Reply:
I’ll move this import, sorry.

Reply:
Almost perfect. There needn't be a newline between the two different ways of doing system imports. Here's the recommendation we follow: https://www.python.org/dev/peps/pep-0008/#imports